### PR TITLE
Enable support for infinispan-operator support for ppc64le(tested on UBI9)

### PR DESCRIPTION
**Pull request by shubham-bhagwat0**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: i/infinispan-operator/infinispan-operator_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: infinispan-operator
+# Version	: 2.4.0.Final
+# Source repo	: https://github.com/infinispan/infinispan-operator.git
+# Tested on	: Red Hat Enterprise Linux 87(8.7) && UBI 9 (9.3)
```
Comment (AbhiIBM) [inline i/infinispan-operator/infinispan-operator_ubi_9.3.sh:6]: Tested on value should be "UBI:9.3" else the script will fail.

File: i/infinispan-operator/infinispan-operator_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+if ! git clone $PACKAGE_URL -b $PACKAGE_BRANCH ; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 0
+fi
+
+# Increase timeout to resolve golangcli-lint timeout error
+/home/go/src/infinispan-operator/bin/golangci-lint run --enable errorlint --timeout=10m
+go mod vendor
+make lint
```
Comment (AbhiIBM) [inline i/infinispan-operator/infinispan-operator_ubi_9.3.sh:64]: Add proper "if fi" here for building package.

File: i/infinispan-operator/infinispan-operator_ubi_9.3.sh [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+if ! make test; then
+	echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_success_but_test_Fails"
+	exit 1
```
Comment (AbhiIBM) [inline i/infinispan-operator/infinispan-operator_ubi_9.3.sh:69]: Exit value should be 2 here.

**Review by shubham-bhagwat0 — COMMENTED** [review]
changes made, fixed suggested issues

**Pull request closed without merging**